utils/gyb_syntax_support/NodeSerializationCodes.py

```diff
@@ -239,7 +239,7 @@
     'DifferentiationParams': 235,
     'DifferentiationParamList': 236,
     'DifferentiationParam': 237,
-    'DifferentiableAttributeFuncSpecifier': 238,
+    # removed: 'DifferentiableAttributeFuncSpecifier': 238,
```

Comment:
minor: I wonder if it's a breaking change to change syntax node serialization codes. If not, decrementing higher serialization codes would be nice, so there's no gap.

@rintaro: could you please provide a suggestion here?

Reply:
I saw other places above that did this, so I imitated those.

Reply:
They are already in `swift-5.2-branch`. So please keep higher codes as is.